### ISSUE TITLE
UnsatisfiableError when installing from conda-forge

### ISSUE THREAD

**Issue by maxhully** [geopandas/geopandas#982]
I am getting an UnsatisfiableError when trying to install geopandas in a fresh conda environment. To replicate:

```
> conda create -n myenv python=3.6
> conda activate myenv
> conda install -c conda-forge geopandas
Collecting package metadata: done
Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - geopandas
Use "conda search <package> --info" to see the dependencies for each package.
```
I am using conda 4.6.12 on Windows 10. Is this a known issue?

**Comment by ocefpaf** [comment]
Please open issue regarding the `conda-forge` package at https://github.com/conda-forge/geopandas-feedstock/issues

Can you follow the instruction [from our docs](https://conda-forge.org/docs/user/tipsandtricks.html#how-to-fix-it) and try again? I was not able to reproduce your problem with the right configuration (`conda-forge` on top of `defaults` and `strict` channel enabled.)

```shell
conda create --name TEST python=3.6 geopandas
conda activate TEST
python -c "import geopandas; print(geopandas.__version__)"
0.4.1
```

**Comment by maxhully** [comment]
I set my configuration as prescribed in the docs and still got the same error. I also ran `conda config --show` and verified that the `channel_priority` was `strict` with `conda-forge` over `defaults`.

**Comment by ocefpaf** [comment]
The lack of a rebuild of `qt` with `openssl`, see https://github.com/conda-forge/qt-feedstock/pull/98, is really hurting us on everything that depends on `qt` (in this case `matplotlib`).

@jorisvandenbossche can we depend on `matplotlib-base` here instead of `matplotlib`? That would allow users to change the backend to agg and use it without waiting on `qt`.

**Comment by jorisvandenbossche** [comment]
> can we depend on matplotlib-base here instead of matplotlib? 

Yes, sure. 
matplotlib is actually only an optional dependency, but I rather keep it in general, as almost everybody will need it (in order to keep descartes, to avoid that users need to install descartes manually)

**Comment by ocefpaf** [comment]
Thanks. I'll give that a try soon. Still, for `geopandas` to work with jupyter (or anything that need `qt`) we need to fix `qt` :-(

The good news is that we already fixed `qt` on macOS and Linux, the bad news is that Windows is quite difficult and we don't have enough Windows experts.

**Comment by numeric-lee** [comment]
Cannot install geopandas to JupyterLab using 
`conda install -c conda-forge/label/cf201901 geopandas`
or
`conda install -c conda-forge/label/geopandas_dev geopandas`

When I install it using
`conda install -c conda-forge/label/cf202003 geopandas`
the install works but the python import fails



```
$ **conda install -c conda-forge/label/cf202003 geopandas**
Collecting package metadata: done
Solving environment: done

# All requested packages already installed.

(lex) le@b4d54aa411df data
$ python
Python 3.6.10 | packaged by conda-forge | (default, Mar  5 2020, 10:05:08)
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import geopandas as gpd
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/le/.conda/envs/lex/lib/python3.6/site-packages/geopandas/__init__.py", line 1, in <module>
    from geopandas.geoseries import GeoSeries  # noqa
  File "/home/le/.conda/envs/lex/lib/python3.6/site-packages/geopandas/geoseries.py", line 15, in <module>
    from geopandas.base import GeoPandasBase, _delegate_property
  File "/home/le/.conda/envs/lex/lib/python3.6/site-packages/geopandas/base.py", line 13, in <module>
    from .array import GeometryArray, GeometryDtype
  File "/home/le/.conda/envs/lex/lib/python3.6/site-packages/geopandas/array.py", line 7, in <module>
    from pandas.api.extensions import ExtensionArray, ExtensionDtype
ModuleNotFoundError: No module named 'pandas.api.extensions'
```
>>>


(lex) le@b4d54aa411df data
$ conda install -c conda-forge/label/geopandas_dev geopandas
Collecting package metadata: done
Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - geoviews
Use "conda search <package> --info" to see the dependencies for each package.

**Comment by martinfleis** [comment]
@numeric-lee standard `conda install -c conda-forge geopandas` works? Without a specification of a label?

**Comment by numeric-lee** [comment]
@martinfleis  Thanks for the prompt response. Unfortunately, no luck there.   
```
(lex) le@b4d54aa411df data
$ conda install -c conda-forge geopandas
Collecting package metadata: done
Solving environment: | Killed
(lex) le@b4d54aa411df data
```
$

**Comment by ocefpaf** [comment]
> Cannot install geopandas to JupyterLab using

We need more information there. JupyterLab is a UI and not something you install packages on, so I'm guessing you have an image or environment with JupyterLab that you want to install geopandas too.

If so, bare in mind that pre-existing packages/channels in your image may be incompatible with the conda-forge geopandas you want to install. I recommend to create an environment from scratch following  https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge. And then:

```
conda create --name MYENV python=3.8 geopandas jupyterlab
```

and add the other packages you need there.

**Comment by ocefpaf** [comment]
@numeric-lee I'm closing this issue b/c the context of your problem is not the same of the original one. Please open a new issue and fill in the required debug information there.

**Comment by martinfleis** [comment]
@ocefpaf I can :). What you said applies to the new issue and also the original one, so closing with a pointer to a `geopandas-feedstock` repo.

**Comment by ocefpaf** [comment]
> @ocefpaf I can :). What you said applies to the new issue and also the original one, so closing with a pointer to a `geopandas-feedstock` repo.

BTW, b/c conda-forge issues on geopandas installation are quite common, maybe we should add the feedstock link and a message on the issue template to guide people there.
